### PR TITLE
Grugslair and more

### PR DESCRIPTION
**Pull request by Rashmi-278**
Add:

- Grug's lair
- Union
- ApeCoin
- BiFi
- ElyFi
- GasDao
- BleepsDAO  [ Only protocol-info will be added, governance link not found ] 
- Braintrust

Note: Treasury address could only be found for Grug's lair, and ApeCoin's treasury addresses were found but they were above 90 addresses https://apecoin.com/treasury-wallets , so didn't add them as it didn't seem right

### CODE REVIEW

File: protocols/bleepsdao/overview.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Bleeps DAO
+
+## Overview
+
+### What are Bleeps ?
```
Comment (cryptosabago) [inline protocols/bleepsdao/overview.md:5]: New line

File: protocols/bleepsdao/overview.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Bleeps use the same pioneer concept introduced in Mandalas.eth, where the metadata is fully generated from the contract, zero externalities, no backend, no ipfs, nor any client code required. While mandalas generated bitmaps, Bleeps generate sounds!
+
+### How to get a Bleep ?
+
+Bleeps are initially mintable through a prelimary sale at 0.1 ETH each Bleep. 75% of the proceeds go to the Bleeps DAO. There is a hard-coded max supply of 1024 sounds but only 576 sounds are currently available made of 9 instrument each composed of 64 notes (frequencies). 128 of them are currently reserved by the Bleeps' creator (wighawag) and might be distributed differently. The rest of the Bleeps (448) do not yet exist but can potentially be created later at the discretion of the DAO.
+
+### What are the benefits of owning a Bleep ?
+Owning a Bleep transforms a mere user into a Bleeper. Bleepers are given vote in the Bleeps DAO and earn a passive revenue from Melodies created from Bleeps they own. They also get to vote on proposal about the DAO treasury which they manage. One Bleep, One Vote.
+
+### Wait... Melodies ?
```
Comment (cryptosabago) [inline protocols/bleepsdao/overview.md:19]: New line

File: protocols/bleepsdao/overview.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+Melodies are the next steps for Bleeps. A Melody is a sequence of 32 Bleeps fine-tuned by their creator to output the most awesome tracks. Melodies are put on auction as they are minted, and the revenue from the auction is split between the Creator (90%), the Bleepers (5%) and the DAO (5%). You can test a demo version [here](https://demo.bleeps.art/create/).
+
+## Governance
```
Comment (cryptosabago) [inline protocols/bleepsdao/overview.md:22]: New line

File: protocols/bleepsdao/overview.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+Like Nouns DAO, Bleeps DAO utilizes a fork of Compound Governance and its main governing body. The Bleep DAO treasury receives a share (5%) of the ETH proceeds from Melodies auctions. Each Bleep is an irrevocable member of the Bleep DAO and entitled to one vote in all governance matters. The DAO has full power over the future of Bleeps, new instruments (up to 16), evolution of Melodies and economic models.
+
+![](https://i.imgur.com/wWdwixD.png)
+
+## Join us
```
Comment (cryptosabago) [inline protocols/bleepsdao/overview.md:27]: New line

File: protocols/bleepsdao/overview.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+## Governance
+Like Nouns DAO, Bleeps DAO utilizes a fork of Compound Governance and its main governing body. The Bleep DAO treasury receives a share (5%) of the ETH proceeds from Melodies auctions. Each Bleep is an irrevocable member of the Bleep DAO and entitled to one vote in all governance matters. The DAO has full power over the future of Bleeps, new instruments (up to 16), evolution of Melodies and economic models.
+
+![](https://i.imgur.com/wWdwixD.png)
+
+## Join us 
+Get in touch with us on [Discord](https://discord.com/invite/DRtq7xBdtn)
+
+## Additional links
```
Comment (cryptosabago) [inline protocols/bleepsdao/overview.md:30]: New line

File: protocols/braintrust/overview.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
+# Braintrust
+
+## Overview
```
Comment (cryptosabago) [inline protocols/braintrust/overview.md:3]: Please put an empty line between subheaders and paragraphs.

File: protocols/braintrust/overview.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+Braintrust is the first decentralized talent network that connects skilled, vetted knowledge workers with the world’s most reputable brands. The community that relies on Braintrust to find work are the same people who own and build it, ensuring the network always serves the needs of its users, instead of a centrally-controlled corporation. Unlike other marketplaces that take 20% to 40% of talent earnings, Braintrust allows talent to keep 100% of earnings and enables clients to make their budgets go further by cutting out the unnecessary middle men.
+The community expands beyond 50,000 members across 100+ countries. Braintrust partners with global enterprises including Nestle, Porsche, Atlassian, Goldman Sachs, and Nike. Braintrust is backed by Tiger Global, Coinbase Ventures, Pantera, and Multicoin Capital along with a contingent of individual investors. 
+
+
+## Governance
+
+### The Token-Enabled BTRST Governance Model
```
Comment (cryptosabago) [inline protocols/braintrust/overview.md:10]: 👀

File: protocols/braintrust/overview.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+
+Jury Members serve a one-year term and are self-volunteered. Another crucial member of the Dispute Resolution Program is the Jury Arbitrator. Jury Arbitrators are elected through the off-chain Braintrust governance process and serve a 6 month term. They do not issue rulings but walk claimants and disputants through the process, organize the claims and counterclaims, recruit eyewitnesses, and randomly assign Jury Members to serve for a specific claim. Learn more about Braintrust dispute resolution here. 
+
+## Join Us
+
+Reach out to us on [Discord](https://discord.com/invite/braintrust) or [Telegram](https://t.me/braintrustofficial)
+
+## Additional links
```
Comment (cryptosabago) [inline protocols/braintrust/overview.md:43]: 👀

File: protocols/union/overview.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
+# Union
+
+## Overview
```
Comment (cryptosabago) [inline protocols/union/overview.md:3]: New lines for all the subheaders in this file too.

**Review by cryptosabago — DISMISSED** [review]

File: protocols/union/overview.md (hunk 9 above)
Comment (Rashmi-278) [reply protocols/union/overview.md:3]: Demo of markdown formatting https://hackmd.io/JSKzs_V0R--TP7bmHstn0w

File: protocols/braintrust/overview.md (hunk 8 above)
Comment (Rashmi-278) [reply protocols/braintrust/overview.md:43]: Demo of markdown formatting https://hackmd.io/JSKzs_V0R--TP7bmHstn0w

File: protocols/braintrust/overview.md (hunk 7 above)
Comment (Rashmi-278) [reply protocols/braintrust/overview.md:10]: Demo of markdown formatting https://hackmd.io/JSKzs_V0R--TP7bmHstn0w

File: protocols/braintrust/overview.md (hunk 6 above)
Comment (Rashmi-278) [reply protocols/braintrust/overview.md:3]: Demo of markdown formatting https://hackmd.io/JSKzs_V0R--TP7bmHstn0w

File: protocols/bleepsdao/overview.md (hunk 5 above)
Comment (Rashmi-278) [reply protocols/bleepsdao/overview.md:30]: Demo of markdown formatting https://hackmd.io/JSKzs_V0R--TP7bmHstn0w

File: protocols/bleepsdao/overview.md (hunk 4 above)
Comment (Rashmi-278) [reply protocols/bleepsdao/overview.md:27]: Demo of markdown formatting https://hackmd.io/JSKzs_V0R--TP7bmHstn0w

File: protocols/bleepsdao/overview.md (hunk 3 above)
Comment (Rashmi-278) [reply protocols/bleepsdao/overview.md:22]: Demo of markdown formatting https://hackmd.io/JSKzs_V0R--TP7bmHstn0w

File: protocols/bleepsdao/overview.md (hunk 2 above)
Comment (Rashmi-278) [reply protocols/bleepsdao/overview.md:19]: Demo of markdown formatting https://hackmd.io/JSKzs_V0R--TP7bmHstn0w

File: protocols/bleepsdao/overview.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+
+![](https://i.imgur.com/QnoImbP.png)
+
+Bleeps use the same pioneer concept introduced in Mandalas.eth, where the metadata is fully generated from the contract, zero externalities, no backend, no ipfs, nor any client code required. While mandalas generated bitmaps, Bleeps generate sounds!
+
+### How to get a Bleep ?
+
+Bleeps are initially mintable through a prelimary sale at 0.1 ETH each Bleep. 75% of the proceeds go to the Bleeps DAO. There is a hard-coded max supply of 1024 sounds but only 576 sounds are currently available made of 9 instrument each composed of 64 notes (frequencies). 128 of them are currently reserved by the Bleeps' creator (wighawag) and might be distributed differently. The rest of the Bleeps (448) do not yet exist but can potentially be created later at the discretion of the DAO.
+
+### What are the benefits of owning a Bleep ?
```
[the remark this answers is not in the document]
Comment (Rashmi-278) [reply protocols/bleepsdao/overview.md:16]: Demo of markdown formatting https://hackmd.io/JSKzs_V0R--TP7bmHstn0w

File: protocols/bleepsdao/overview.md (hunk 1 above)
Comment (Rashmi-278) [reply protocols/bleepsdao/overview.md:5]: Demo of markdown formatting https://hackmd.io/JSKzs_V0R--TP7bmHstn0w

**Review by CarlGocht — APPROVED** [review]

**Pull request merged**